binaries/runtime/Cargo.toml

```
@@ -37,6 +37,7 @@ clap = { version = "4.0.3", features = ["derive"] }
 tracing-opentelemetry = { version = "0.18.0", optional = true }
 pythonize = { version = "0.19.0", optional = true }
 arrow-schema = "45.0.0"
+arrow = { version = "45.0.0", features = ["pyarrow"] }
```

```suggestion
arrow = { version = "45.0.0" }

// ...

python = ["pyo3", "dora-operator-api-python", "pythonize", "arrow/pyarrow"]
```

To avoid pyo3 static library linking error

---

Could you clarify what link error you mean? I don't see any errors locally or on CI.

---

Ok, so basically, if you use pre-compiled dora-runtime that is shipped with dora-daemon, the dora-daemon is going to link pythonlib because arrow depend on pyarrow. 

However it is going to use the one from the source environment instead of the runtime environment, making `cannot find pythonlib`errors. 

But the thing is we only need to link to python if we're using the runtime in python. I'm going to edit the suggestion to make it clearer

---

Ah, of course, thanks for clarifying!

---

Applied in 5b9a0828.